tests/integration/api/test_api.py

```diff
@@ -24,18 +24,27 @@ def test_api_schema(case: Case, app: FastAPI) -> None:
         case.call_and_validate(session=session)
 
 
-def test_get(
+def test_get_default_list_metadata(populated_app: FastAPI) -> None:
```

Comment:
Please remove this test and replace it one that tests something of value. Assertions should always be used to further increase the confidence in a system and the value it is delivering. Asserting that an endpoint exists and returns a 200 status code doesn't tell the developer running the test anything that they couldn't have otherwise guessed.